common/theme.c

```diff
@@ -123,3 +135,64 @@ void themeStartup(ThemePreset preset) {
     }
     config_destroy(&cfg);
 }
+
+void GetThemePathFromConfig(char* themePath) {
+    const char* tmpThemePath = "";
+    config_t cfg = {0};
+    config_setting_t *setting;
+    char tmp_path[PATH_MAX] = {0};
+
+    #ifdef __SWITCH__
+    tmp_path[0] = '/';
+    #endif
+
+    strncat(tmp_path, "config/nx-hbmenu/settings.cfg", sizeof(tmp_path)-2);
+    bool good_cfg = config_read_file(&cfg, tmp_path);
+    
+    if(good_cfg) {
+        setting = config_lookup(&cfg, "hbmenuConfig");
```

Comment:
settings, and see eariler comment regarding that string name.

Reply:
done